setup.py

```diff
@@ -46,6 +46,7 @@ def run_tests(self):
         'presto': ['requests>=1.0.0'],
         'trino': ['requests>=1.0.0'],
         'hive': ['sasl>=0.2.1', 'thrift>=0.10.0', 'thrift_sasl>=0.1.0'],
+        'hive_pure': ['pure-sasl>=0.6.2', 'thrift>=0.10.0', 'thrift_sasl>=0.1.0'],
```

Comment:
s/hive_pure/hive_pure_sasl - to be more specific about the difference

Reply:
done